items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -22,7 +24,11 @@ import scala.concurrent.{ExecutionContext, Future}
   *  can be placed on an item.
   *
   */
-class SierraItemUpdater(sierraSource: SierraSource)(
+class SierraItemUpdater(
+  sierraSource: SierraSource,
+  venueOpeningTimesLookup: VenueOpeningTimesLookup,
+  clock: Clock
```

Comment:
```suggestion
  venueClock: Clock
```
Agreed on using a `Clock` but still important not to lose context of what it's doing at the point of use